File: Code.c
#include <stdio.h>

int main(){
int a;
int b;
int c;
float d;
printf("Programa Teste \n");
printf("Digite A: ");
scanf("%d", &a);
printf("Digite B: ");
scanf("%d", &b);
if (a<b){
c=a+b;

}else {
c=a-b;

}printf("C eh igual a: ");
printf("%d\n",c);
d=c/(a+b);
printf("D e igual a: ");
printf("%f\n",d);

}

File: Projeto-Compiladores-main/Code.py
def main():
	a = int()
	b = int()
	c = int()
	d = float()
	print("Programa Teste \n")
	print("Digite A: ")
	a = int(input()) 
	print("Digite B: ")
	b = int(input()) 
	if  a<b:
		c=a+b

	else :
		c=a-b

	print("C eh igual a: ")
	print("%d" %c)
	d=c/(a+b)
	print("D e igual a: ")
	print("%f" %d)


main()
